Full-Stack-Collective/connectient
# Configure confirm email

## Description

The request and confirmation emails will now show the correct practice name and info based on the practice ID that is assigned to the patient. 

## Related Issue

closes #106 

## Acceptance Criteria

Practice information in emails should correctly match the practice that is associated with the patient receiving the emails.

## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|     | :bug: Bug fix              |
|    | :sparkles: New feature     |
| ✓  | :hammer: Refactoring       |
|     | :100: Add tests            |
|     | :link: Update dependencies |
|     | :scroll: Docs              |

## Updates

### Before
![Screen Shot 2023-09-14 at 5 20 05 PM](https://github.com/Full-Stack-Collective/connectient/assets/70449557/c7d9eaf8-c482-41b3-a640-a257727d7dbd)

### After
![Screen Shot 2023-09-14 at 5 06 07 PM](https://github.com/Full-Stack-Collective/connectient/assets/70449557/f9d1c5c2-e614-4090-a1c1-45926f977161)

## Testing Steps / QA Criteria

Submit an appointment request, entering your email in as the email for the patient. As long as your email is set in the .env.local file as EMAIL, then you should receive an email with the correct name, address, website link, and image of the practice the appointment was created for as well as the request data. If you then log into as an admin for that practice and schedule that request, you should receive another email that also contains the correct practice info. Note: If you try and confirm older appointments that we created they may not have all the correct data available and therefore some email data will come up as 'undefined'.
